test/Feature/SingularSpec.hs

```diff
@@ -56,7 +56,7 @@ spec =
         _ <- post "/addresses" [json| { id: 98, address: "xxx" } |]
         _ <- post "/addresses" [json| { id: 99, address: "yyy" } |]
         p <- request methodPatch "/addresses?id=gt.0"
-                [("Prefer", "return=representation"), singular]
+                [singular]
```

Comment:
It'd be better to not modify previous tests that have both `Prefer:return=representation` and `singular`. Instead you could add new tests for POST/PATCH/DELETE that only have the `singular` header and fail with 406.